### ISSUE TITLE
[New device support]: TONGOU ‎TO-Q-SY1-ZT - Smart DIN Rail Switch

### ISSUE THREAD

**Issue by mklooss** [Koenkk/zigbee2mqtt#20892]
### Link

https://www.amazon.de/dp/B0CHVF7D4N

### Database entry

{"id":81,"type":"EndDevice","ieeeAddr":"0xa4c1385d3a953a12","nwkAddr":56860,"manufId":4417,"manufName":"_TZ3000_gzvniqjb","powerSource":"Mains (single phase)","modelId":"TS0011","epList":[1],"endpoints":{"1":{"profId":260,"epId":1,"devId":256,"inClusterList":[3,4,5,6,0],"outClusterList":[25,10],"clusters":{"genBasic":{"attributes":{"65503":"\u0001\u0000\u0000\u0000\u0011\u0000\u0000\u0000\u0000\u0011","65506":55,"65508":0,"65534":0,"modelId":"TS0011","manufacturerName":"_TZ3000_gzvniqjb","powerSource":3,"zclVersion":3,"appVersion":80,"stackVersion":0,"hwVersion":1,"dateCode":""}},"genOnOff":{"attributes":{"onOff":0,"onTime":0,"offWaitTime":0,"tuyaBacklightMode":1,"moesStartUpOnOff":2}}},"binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0x00124b00214f1fad","endpointID":1}],"configuredReportings":[],"meta":{}}},"appVersion":80,"stackVersion":0,"hwVersion":1,"dateCode":"","zclVersion":3,"interviewCompleted":true,"meta":{"configured":1071524409},"lastSeen":1705669676089,"defaultSendRequestWhen":"immediate"}

### Comments

Similar to: https://www.zigbee2mqtt.io/devices/TO-Q-SY1-JZT.html

See also:
https://github.com/dresden-elektronik/deconz-rest-plugin/pull/6842
https://zigbee.blakadder.com/Tongou_TO-Q-SY1-ZT.html

### External defintion

```shell
const {onOff} = require('zigbee-herdsman-converters/lib/modernExtend');

const definition = {
    zigbeeModel: ['TS0011'],
    model: 'TS0011',
    vendor: '_TZ3000_gzvniqjb',
    description: 'Automatically generated definition',
    extend: [onOff({powerOnBehavior: false})],
};

module.exports = definition;
```

**Comment by Trebor87** [comment]
Hello

I have just bought this device, It installs ok but canot OTA update it, any idea when this will be possible ?

Many thanks

**Comment by Mark-64** [comment]
Hi, I also I bought this device and it is recognized as Tuya [TS0011](https://www.zigbee2mqtt.io/devices/TS0011.html#tuya-ts0011).
It works ok as a switch bit it doesn't seem to have Zigbee router capability, not sure if this is correct, as the similar model TO-Q-SY1-JZT (with power monitoring) can behave as a router.

**Comment by niikkoo** [comment]
@Koenkk : 
Hello  ,  Will you take in charge this  device support  or it is a lost time  
currently this devie is still detected as  Tuya [TS0011](https://www.zigbee2mqtt.io/devices/TS0011.html#tuya-ts0011).
Feel free to knock me for any additional infos 
thank you for your time and your work

**Comment by Koenkk** [comment]
The TO-Q-SY1-JZT is a different device, it has Zigbee model ID `TS011F`, this one has `TS0011`. I also don't see any electrical measurements cluster being supported, so I'm wondering if this device can do power measurements at all. Can someone check if this work with the original gateway?

**Comment by niikkoo** [comment]
Thank you both for your answers  :)
i finally  ordered black version TO-Q-SY2-Jwt , hopping it will detect as expected

**Comment by oscarpuigcerver** [comment]
I have 3 new TO-Q-SY2-163JZT but in all cases is showing as unsupported, someone else with the same issue?

**Comment by djereg** [comment]
I bought a TO-Q-SY1-ZT. It is reconized by Z2M as _TZ3000_gzvniqjb and a TS0011 model.

Displayed as a regular 1 gang wall switch, but it's ok, since technically it's a regular switch (without metering).
But, it is recognized as an end device and displaying unknown battery state.

I dont know does it works as a router, but Z2M does not recognize it as a router.

![image](https://github.com/user-attachments/assets/9fb10dcc-87bd-4da4-8f19-d8e9c81fa942)

**Comment by dorianmartinez31** [comment]
Does anyone worked on TO-Q-SY1-ZT integration?
I assume copying TO-Q-SY1-JZT and removing any exposed features related to metering should work.
Is anyone capable to do this?

**Comment by preussal** [comment]
> Hello, Does anyone worked on TO-Q-SY1-ZT integration? I assume copying TO-Q-SY1-JZT and removing any exposed features related to metering should work. Is anyone capable to do this?

Hi, I have the TO-Q-SY1-JZT via Zigbee2MQtt in home assistant. 
Works quite well so far. 
Voltage is displayed immediately. 
Only the Power and Current display is a bit slow. It takes sometimes forever to show what is the Power usage when you switch on.
Switching is immediately, only the power measurement is slow

**Comment by SaaX-IRL** [comment]
Hiya. I've just ordered the TO-Q-SY1-ZT and was wondering how to get this integrated? I'm esp. interested in the ability to configure the over_voltage_setting.

**Comment by SaaX-IRL** [comment]
@mklooss yeah, I understand that, but I need to increase the overvoltage threshold more than 255V (it's 269V in IE). As far as I understand that can't be done (yet) for the TO-Q-SY1-ZT. Maybe I'm just misunderstanding?

Is there any way of "applying" the TO-Q-SY1-JZT exposed settings to the TO-Q-SY1-ZT, like do I have any choice or is it forced by discovery?

Sorry, I'm a total noob when it comes to Z2M, just a user. Any advice would be highly appreciated.

**Comment by muchasuerte** [comment]
do you know it the device support on_time or off_wait_time ? I did several test with no success
